.github/workflows/release.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Test (changed packages)
         run: lerna run test --concurrency 3 --since HEAD~
       - name: Build (changed packages)
-        run: lerna run build --concurrency 3 --since HEAD~
```

Comment:
Not sure where this change come from, see `next`:

https://github.com/nextauthjs/adapters/blob/5c915823c43867dd54adf3977c26968baa5ec11f/.github/workflows/release.yml#L33

Could you maybe update your branch by merging `next` into it?

Reply:
It's odd, I already merged from next. And merged agin and this change isn't in the merge. 

I don't know why, so I made the edit manually.